hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/util/HiveSchemaUtil.java

```diff
@@ -446,58 +448,76 @@ public static String generateSchemaString(MessageType storageSchema, List<String
     return generateSchemaString(storageSchema, colsToSkip, false);
   }
 
-  public static String generateSchemaString(MessageType storageSchema, List<String> colsToSkip, boolean supportTimestamp) throws IOException {
+  /**
+   * Generates the Column DDL string for creating a Hive table from the given schema.
```

Comment:
I love the original document that we add a SQL statement demo for the method, can we keep that?